### PR TITLE
Update Social Links and APIs

### PR DESCRIPTION
**Pull request by nzec**
* Added Flickr API and Twitter API with FOSSASIA account.
* Added FOSSASIA Social and Web Links
* Updated FontAwesome (Fonts as well as CSS)
* Added fossasia-light.png

### CODE REVIEW

File: index.html
Comment (abishekvashok) [inline index.html]: These are required for jekyll.

**Comment by abishekvashok** [comment]
Typo in the PR title

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by abishekvashok** [comment]
Can you double check, the icons are not rendering.
screenshot from 2017-12-01 13-29-21

File: index.html
Comment (nzec) [reply index.html]: @abishekvashok Oh okay. And as for the icons not rendering, please let it load completely.

Comment (nzec) [reply index.html]: @abishekvashok Fixed.

**Comment by abishekvashok** [comment]
Still the same @duttaditya18 

screenshot from 2017-12-01 13-36-35

**Comment by nzec** [comment]
@abishekvashok I fixed it. Check it out now.

**Comment by harshit98** [comment]
@duttaditya18 Kindly provide screenshot of the fix. Don't hurry. Take your time. :)
Keep it up!

**Review by abishekvashok — APPROVED** [review]
LGTM at: https://duttaditya18.github.io/gci17.fossasia.org/

**Comment by CosmicCoder96** [comment]
@duttaditya18 Again, please squash the commits :)

**Review by abhinavk96 — CHANGES_REQUESTED** [review]
@duttaditya18 Again, please squash the commits :)

File: index.html
Comment (abhinavk96) [inline index.html]: @duttaditya18 that's not how it's supposed to be, please check.

**Review by abhinavk96 — CHANGES_REQUESTED** [review]

**Comment by nzec** [comment]
@harshit98 The Flickr fix does not work on Github Pages because the API is synced with the gci17.fossasia.org website. You need to download it to see. It will work when you put it up. 

@CosmicCoder96 I do not know how to and I messed up the previous repo by trying to do so from GitKRaken.

And here are the screenshots : https://imgur.com/a/xdrJn

File: index.html
Comment (nzec) [reply index.html]: Fixed.

**Comment by abhinavk96** [comment]
@duttaditya18 Please use git command line instead of git GUI gitKraken etc. There are plenty of resources available to learn how to squash commits on git. try.github.io is where you should start :)

**Comment by nzec** [comment]
@CosmicCoder96 Okay. So I have merged my commits. So can now this be merged?

**Comment by harshit98** [comment]
@duttaditya18 The pull request is still showing 3 commits. Is there any issue you're facing ? Let me know.

**Comment by abhinavk96** [comment]
@harshit98 no, the commits are squashed now. Refresh.

**Review by harshit98 — APPROVED** [review]

**Review by abhinavk96 — APPROVED** [review]
LGTM!

**Comment by nzec** [comment]
@CosmicCoder96 I had to resolve the conflicts, so there had to be 2 more commits.

**Comment by nzec** [comment]
Just merge this already!

**Comment by jigyasa-grover** [comment]
@duttaditya18  Kindly be patient and perhaps squash your 5 commits into 1 ? :)

**Comment by nzec** [comment]
Okay @jigyasa-grover If I resolve this conflict, there will be another commit. Can't this be just merged already?

**Comment by abishekvashok** [comment]
@jigyasa-grover since, @duttaditya18 is new, can we use the squash and merge button after he resolves the conflicts?

**Comment by nzec** [comment]
@abishekvashok @jigyasa-grover I have resolved all the conflicts. It would be awesome if this is merged.

**Comment by jigyasa-grover** [comment]
@duttaditya18 Resolve the conflicts pls, we'd "squash & merge", thanks !

**Comment by nzec** [comment]
@jigyasa-grover I have resolved all the conflicts. Merge this before more conflicts arise.

**Comment by nzec** [comment]
@jigyasa-grover @abishekvashok So will this be merged now? Like please. Just do it already.

**Comment by abishekvashok** [comment]
@duttaditya18 please wait, I don't have write access, let's wait for a someone with write access to squash and merge this.

Ping @fossasia/fossasia-gci

**Review by madhavrathi — APPROVED** [review]

**Comment by nzec** [comment]
Link to my task: https://codein.withgoogle.com/dashboard/task-instances/6633250158018560/

**Pull request merged**
